src/storages/s3.py

```
@@ -0,0 +1,80 @@
+from typing import Union
+
+import aioboto3
+from botocore.exceptions import (
+    BotoCoreError,
+    NoCredentialsError,
+    HTTPClientError,
+    ConnectionError
+)
```

Pylint (3.3.7)

[warning] Redefining built-in 'ConnectionError'

(W0622)